cves/CVE-2017-5664.yml

```diff
@@ -212,4 +220,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Testing is very important when developing software. Edge cases need to be 
+    tested. This includes error handling cases. 
```

Comment:
The explanation is too generic. You need to explain how testing would apply to this CVE specifically. How would testing improve?